### DISCUSSION TITLE
How to export the information of the IPF?

### DISCUSSION THREAD

**Discussion by louis96108** [mtex-toolbox/mtex#1811] [Ask Anything]
Hi, how to export the information of the IPF as the OIM software does?

![tmp1C9E](https://user-images.githubusercontent.com/69557894/236396205-ca0a55fa-8c44-4799-a13c-38b677cfbebc.png)

**Comment by ralfHielscher** [comment]
Some comments

(1) assume you of an `ipf` in MTEX you can evaluate it at any grid by
```matlab
directions = plotS2Grid('resolution',1*degree,ipf.CS.fundamentalSector);
prop.value = ipf.eval(h);
```
(2) now you can use the function [`export`](https://mtex-toolbox.github.io/vector3d.export.html) to write these values to a file, i.e.,
```matlab
directions.export('myfile.txt',prop,'polar')
```
I hope this helps,

Ralf

**Reply by louis96108** [reply]
Thank you so much, I'll try it
